### PR TITLE
CHANGELOG.md for moler 1.9.0

### PR DESCRIPTION
**Pull request by marcin-usielski**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,22 @@
...
+* Iperf2 command fails on parsing when used `with -s -P 1` options (single run server)
+
+## moler 1.9.0
+
+### Added
+* New method of converting string to int for whole dict.
+
+### Improved
+* Keep state in background does not block main flow
+* Timestamp is passed to data_received
```
Comment (greg-latuszek) [inline CHANGELOG.md]: `Timestamp is passed to data_received` --> data_received() of commands & events gets additional parameter: time when data was caught on connection

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,22 @@
...
+* Improvement parsing in command ls
+* Pack observers in separate threads.
```
Comment (greg-latuszek) [inline CHANGELOG.md]: `Pack observers in separate threads` --> `Commands & events data processing is done in separate thread to secure connection reads against blocking them by commands/events`

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,22 @@
+## moler 1.10.0
+
+### Improved
+* Classes Connection and ObservableConnection are deprecated now.
```
Comment (greg-latuszek) [inline CHANGELOG.md]: add to this description: `Their original code has been moved into AbstractMolerConnection and ThreadedMolerConnection. Change made for code clarity.`

File: CHANGELOG.md (hunk 2 above)
Comment (marcin-usielski) [reply CHANGELOG.md]: Added info also about callbacks.

File: CHANGELOG.md (hunk 1 above)
Comment (marcin-usielski) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md (hunk 3 above)
Comment (marcin-usielski) [reply CHANGELOG.md]: Done.

**Pull request merged**